### ISSUE TITLE
[Suggestion] Provide a bin/ruby2js or another bin/ equivalent

### ISSUE THREAD

**Issue by rubyFeedback** [ruby2js/ruby2js#145]
I just download and extracted the gem but noticed no bin/ directory. Would it
be  possible to add such a commandline variant aka bin/ruby2js or something
like that? I can create one on my own (it's just a file anyway and I can put
the necessary require, and code in to evaluate what ruby2js would normally
do) but having this ready-made as-is would be super-convenient. Thank
you for reading this!

**Comment by rubys** [comment]
Fixed by https://github.com/ruby2js/ruby2js/commit/cc39ce46f6210532021f8c72416891b6edda69a1.  Should be released within a week or two.  Meanwhile:

```
git clone https://github.com/ruby2js/ruby2js.git
ruby ruby2js/demo/ruby2js.rb --help
```

**Comment by rubys** [comment]
Now released - v4.2.0
